src/inausoft.netCLI/inausoft.netCLI.csproj

```diff
@@ -7,7 +7,7 @@
     <PackageProjectUrl>https://github.com/inausoft/netCLI</PackageProjectUrl>
     <RepositoryUrl>https://github.com/inausoft/netCLI</RepositoryUrl>
     <PackageTags>CLI command line parsing</PackageTags>
-    <Version>1.0.0-rc.4</Version>
+    <Version>1.0.0-rc.8</Version>
```

Comment:
This version should be 1.0.0 since it will be merged to main

Reply:
Agree.